defold-spine/src/script_spine_gui.cpp

```diff
@@ -101,6 +101,9 @@ namespace dmSpine
      * `playback_rate`
      * : [type:number] The rate with which the animation will be played. Must be positive
      *
+     * `track`
+     * : [type:number] The track to play the animation on. Defaults to track 1.
+     *
```

Comment:
It's not fully functional tracks like in `go`, tracks require much more changes (). But I added it here to make it possible to similar example we have in the `go` world